### PR TITLE
create lecture on development environments

### PR DESCRIPTION
**Pull request by spulick**
- created new lecture (workspace.md)
- added lecture specific graphics
- moved Git content (Exercise 2.2) from getting_started.md to workspace.md

### CODE REVIEW

File: lectures/workspace.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+
+```{contents} Contents
+:depth: 2
+```
+## Overview
+
+In this lecture, you will learn to
+- work with Python scripts
+- set up different development environments
```
Comment (jstac) [inline lectures/workspace.md]: different -> multiple?

File: lectures/workspace.md [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+- get started with GitHub
+
+```{note}
+Going forward, it is assumed that you have an Anaconda environment up and running.
+```
+
+You may want to create a new conda environment if you haven't done so already.
```
Comment (jstac) [inline lectures/workspace.md]: [create a new conda environment](link to conda documentation for this task)

File: lectures/workspace.md [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+
+## Working with Python Files
+
+So far, you have explored the use of Jupyter notebooks in writing and executing Python code.
```
Comment (jstac) [inline lectures/workspace.md]: you -> we

("you" is fine but "we" is more of a convention of the lectures)

File: lectures/workspace.md [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+Going forward, it is assumed that you have an Anaconda environment up and running.
+```
+
+You may want to create a new conda environment if you haven't done so already.
+
+## Working with Python Files
+
+So far, you have explored the use of Jupyter notebooks in writing and executing Python code. 
+
+While they are efficient and adaptable when working with short pieces of code, they are not the best choice for longer programs and scripts.
```
Comment (jstac) [inline lectures/workspace.md]: they -> notebooks

**Comment by jstac** [comment]
Great work @orectique , very nicely written.

Please add this lecture to the table of contents in https://github.com/QuantEcon/lecture-python-programming.myst/blob/main/lectures/_toc.yml after `python_oop`.

Please also see the comments above.  And please add just a little more detail to the overview, mentioning that the lecture will look at VS code and Jupyter Lab specifically.

I would also consider shifting the sentences "So far..." and "While..." just after the Overview section into the Overview section, so that the reader immediately understands the purpose of the lecture -- until now we've been using Jupyter notebooks, but they are inherently limited and as we start to write more code we need more options.

**Comment by spulick** [comment]
Greetings, @jstac 

I have made pushed the recommended changes. I've also added a couple of lines about the use of Jupyter Notebooks in VS Code.

**Pull request closed without merging**